### PR TITLE
Bug/(#640):- Service Account credentials exposed on errors from JIRA client library in the API Response.

### PR DESCRIPTION
**Pull request by anjneeksharma**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

Service account credentials are exposed in any error that comes from jira.

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>
image

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by sayak-sarkar** [comment]
@anjneeksharma Please fix the failing Code scan: [https://github.com/1-Platform/one-platform/security/code-scanning/6?query=ref%3Arefs%2Fpull%2F719%2Fhead](https://github.com/1-Platform/one-platform/security/code-scanning/6?query=ref%3Arefs%2Fpull%2F719%2Fhead)

File: packages/feedback-service/src/resolver.ts [hunk 1]
```diff
@@ -141,6 +141,7 @@ export const FeedbackResolver = {
           } );
         } )
         .catch( ( err: any ) => {
+          err.options.auth = '*******';
```
Comment (ghanlohar) [inline packages/feedback-service/src/resolver.ts:144]: Do we really want this entire error object(err) to transferred to the front end? Can we just extract error message out of it and transferred that only?

Comment (anjneeksharma) [reply packages/feedback-service/src/resolver.ts]: Updated as per suggestion.

**Review by ghanlohar — APPROVED** [review]

**Review by riginoommen — COMMENTED** [review]
Does the error message has error code

**Comment by anjneeksharma** [comment]
@riginoommen  yes that error message has error code.

**Review by riginoommen — APPROVED** [review]

**Pull request merged**